package/CHANGELOG

```diff
@@ -175,6 +177,10 @@ Enhancements
     checking if it can be used in parallel analysis. (Issue #2996, PR #2950)
 
 Changes
+  * The ParmEd classes were moved to the `converters` module (PR #2882)
```

Comment:
Make clear that we now have converters in their own top level module
```suggestion
  * Introduce new converter API with all converters in MDAnalysis.converters (#2790) 
  * The ParmEd classes were moved to the `converters` module (PR #2882)
```

Reply:
Sounds good @cbouy can you implement the change (keeping the PR number as you propose)?

Reply:
Done - needed to restart CI anyways